snap/snapcraft.yaml

```
@@ -23,11 +28,18 @@ apps:
       - network-bind
 
 parts:
-  pollen:
-    plugin: go
+  aproxy:
+    plugin: nil
     source: .
     build-snaps:
       - go
+    override-build: |
+      snapcraftctl build
+      GOARCH=$SNAPCRAFT_TARGET_ARCH
+      CGO_ENABLED=0
+      go mod download
+      go build -ldflags="-w -s"
+      cp aproxy ${SNAPCRAFT_PART_INSTALL}/bin
```

```suggestion
      mkdir ${SNAPCRAFT_PART_INSTALL}/bin
      cp aproxy ${SNAPCRAFT_PART_INSTALL}/bin/aproxy
```
Sorry, this should fix the issue.

---

Thanks for the prompt fix! Updated!